cpp/unreal_plugins/SpEngine/Source/SpEngine/GameWorldService.h

```diff
@@ -6,27 +6,208 @@
+    LOAD_RegenerateBulkDataGuids     = static_cast<std::underlying_type_t<ELoadFlags>>(ELoadFlags::LOAD_RegenerateBulkDataGuids),
+    LOAD_DisableDependencyPreloading = static_cast<std::underlying_type_t<ELoadFlags>>(ELoadFlags::LOAD_DisableDependencyPreloading),
+    LOAD_Quiet                       = static_cast<std::underlying_type_t<ELoadFlags>>(ELoadFlags::LOAD_Quiet),
+    LOAD_FindIfFail                  = static_cast<std::underlying_type_t<ELoadFlags>>(ELoadFlags::LOAD_FindIfFail),
+    LOAD_MemoryReader                = static_cast<std::underlying_type_t<ELoadFlags>>(ELoadFlags::LOAD_MemoryReader),
+    LOAD_NoRedirects                 = static_cast<std::underlying_type_t<ELoadFlags>>(ELoadFlags::LOAD_NoRedirects),
+    LOAD_ForDiff                     = static_cast<std::underlying_type_t<ELoadFlags>>(ELoadFlags::LOAD_ForDiff),
+    LOAD_PackageForPIE               = static_cast<std::underlying_type_t<ELoadFlags>>(ELoadFlags::LOAD_PackageForPIE),
+    LOAD_DeferDependencyLoads        = static_cast<std::underlying_type_t<ELoadFlags>>(ELoadFlags::LOAD_DeferDependencyLoads),
+    LOAD_ForFileDiff                 = static_cast<std::underlying_type_t<ELoadFlags>>(ELoadFlags::LOAD_ForFileDiff),
+    LOAD_DisableCompileOnLoad        = static_cast<std::underlying_type_t<ELoadFlags>>(ELoadFlags::LOAD_DisableCompileOnLoad),
+    LOAD_DisableEngineVersionChecks  = static_cast<std::underlying_type_t<ELoadFlags>>(ELoadFlags::LOAD_DisableEngineVersionChecks)
+};
+
+// This struct is intended to be a wrapper for ESpIncludeSuperFlag declared above. Wrapping enums in a struct with USTRUCT() macro
+// helps us take advantage of UnrealObj and UnrealObjUtils to pass enums to and from Python as a JSON string.
+USTRUCT()
+struct FSpIncludeSuperFlag
+{
+    GENERATED_BODY()
+
+    UPROPERTY()
+    ESpIncludeSuperFlag Enum;
+};
+
+// This struct is intended to be a wrapper for ESpObjectFlags declared above. Wrapping enums in a struct with USTRUCT() macro
+// helps us take advantage of UnrealObj and UnrealObjUtils to pass enums to and from Python as a JSON string.USTRUCT()
+USTRUCT()
+struct FSpObjectFlags
+{
+    GENERATED_BODY()
+
+    UPROPERTY()
+    ESpObjectFlags Enum;
+};
+
+// This struct is intended to be a wrapper for ESpLoadFlags declared above. Wrapping enums in a struct with USTRUCT() macro
+// helps us take advantage of UnrealObj and UnrealObjUtils to pass enums to and from Python as a JSON string.USTRUCT()
+USTRUCT()
+struct FSpLoadFlags
+{
+    GENERATED_BODY()
+
+    UPROPERTY()
+    ESpLoadFlags Enum;
+};
+
+// This struct is intended to be identical to Unreal's FActorSpawnParameters struct, except we tag our struct
+// as a USTRUCT so it is easier to pass to and from Python as a JSON string, see:
+//     Engine/Source/Runtime/Engine/Classes/Engine/World.h
+USTRUCT()
+struct FSpActorSpawnParameters
+{
+    GENERATED_BODY();
+
+    FSpActorSpawnParameters()
```

Comment:
Can we do this instead? The following is more more consistent with our existing style. Also I get errors when I have a `;` after `GENERATED_BODY()`. Does that not happen for you?

```
USTRUCT()
struct FSpActorSpawnParameters
{
    GENERATED_BODY()

    UPROPERTY()
    FName Name = Name_None;

    UPROPERTY()
    AActor* Template = nullptr;
}

Reply:
It is weird - I did not get any errors when I used `;` beside `GENERATED_BODY()`.

Reply:
Oh, weird, this only seems to happen for `UCLASS` types. I tried to add `;` to `SpCoreActor` and I got the following error:

```
/Users/mroberts/code/github/spear/cpp/unreal_projects/SpearSim/Plugins/SpCore/Source/SpCore/SpCoreActor.h(79): Error: Extra ';' before 'public'
```

But I don't get any errors when I added this `USTRUCT` to the same file, even though the keyword `public` is in the exact same location relative to the `;`. 

```
USTRUCT()
struct FSpEnum
{
    GENERATED_BODY()
public:
    UPROPERTY()
    ESpEnum SpEnum;
};
```

Weird. I propose that we always avoid `GENERATED_BODY();` for consistency, so we don't need to remember this weird inconsistent compile-time behavior.